src/main/thrift/parquet.thrift

```diff
@@ -576,5 +631,8 @@ struct FileMetaData {
    * e.g. impala version 1.0 (build 6cf94d29b2b7115df4de2c06e2ab4326d721eb55)
    **/
   6: optional string created_by
+
+  /** Sort order used for each column in this file */
+  7: optional list<ColumnOrder> column_orders;
```

Comment:
There is one `ColumnOrder` for each column, in the same order as the column chunks (which is required to match the order in the schema).